pages/catalog/CatalogHandler.inc.php

```diff
@@ -88,9 +88,13 @@ public function page($args, $request, $isFirstPage = false)
 
         $this->_setupPaginationTemplate($request, count($submissionsIterator), $page, $count, $offset, $total);
 
+        $seriesDao = DAORegistry::getDAO('SeriesDAO'); /* @var $seriesDao SeriesDAO */
+        $seriesIterator = $seriesDao->getByContextId($context->getId());
```

Comment:
It occurred to me that we probably only want to show series which have at least one monograph published in them. Can you filter this list to exclude series with no published items?